### DISCUSSION TITLE
What to do if there is a closed and locked issue, which is my unsolved issue?

### DISCUSSION THREAD

**Discussion by Wastus** [microsoft/vscode-cpptools#6969] [Q&A] [answered]
I've searched through the issue list and found an issue, which describes exactly the issue I'm facing, but it was closed by the author as a duplicate (which in my opinion isn't) and then locked by the bot.

Should I open another issue? Is there a way to unlock an issue?

The issue I'm talking about is #1842.

**Comment by bobbrow** [comment, marked as answer]
We encourage you to open a new issue.  We lock old issues after they've been closed for a while because 9 times out of 10 any new posts on them are actually new issues.

Thanks for the question!

**Comment by sean-mcmanus** [comment]
@Wastus Issue https://github.com/microsoft/vscode-cpptools/issues/1842 is old, so although the symptom may match what you're seeing, the root cause of the symptom is most likely unrelated, and we'd probably need more repro details because forced includes should work with compile_commands.json (in general) -- it's possible other command line arguments that we're not handling could be leading to the failure (1.2.1 fixed a couple such cases).
